Oracle/PCORI_MEDS_SCHEMA_CHANGE_ora.sql

```diff
@@ -16,20 +56,22 @@ cui(c_fullname,pcori_cui,c_hlevel) as
     select c_fullname,pcori_cui,c_hlevel from pcornet_med where pcori_cui is not null
    union all
     select m.c_fullname, cui.pcori_cui, m.c_hlevel
-    from pcornet_med m inner join cui on cui.c_fullname=m.c_path where m.pcori_cui is null
+    from I2B2METADATA.pcornet_med m inner join cui on cui.c_fullname=m.c_path where m.pcori_cui is null
 ),
 cuid as
 ( select c_fullname, pcori_cui, row_number() over (partition by C_FULLNAME order by c_hlevel desc)as rowno  from cui)
  
 select DISTINCT cuid.c_fullname as c_fullname, cuid.pcori_cui as pcori_cui from cuid where cuid.rowno=1
 );
- 
+
+-- TODO: work on performance
```

Comment:
Are you having a performance problem? This script shouldn't take more than a minute to run.